dojo/settings/settings.dist.py

```diff
@@ -1250,7 +1250,7 @@ def saml2_attrib_map_format(dict):
     'StackHawk HawkScan': ['vuln_id_from_tool', 'component_name', 'component_version'],
     'Hydra Scan': ['title', 'description'],
     'DrHeader JSON Importer': ['title', 'description'],
-    'PWN SAST': ['title', 'description'],
+    'PWN SAST': ['unique_id_from_tool'],
```

Comment:
You also want to swith from DEDUPE_ALGO_HASH_CODE to DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL

Reply:
All set.  Thanks @manuel-sommer